server/src/main/java/com/iqas/server/service/DocumentService.java

```diff
@@ -0,0 +1,52 @@
+package com.iqas.server.service;
+
+
+import com.iqas.server.model.DocumentModel;
+import com.iqas.server.model.KbModel;
+import com.iqas.server.repository.DocumentRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@Service
+public class DocumentService {
+
+    @Autowired
+    private DocumentRepository documentRepository;
+
+    @Autowired
+    private NextSequenceService nextSequenceService;
+
+    @Autowired
+    private KbService kbService;
+
+    private List<DocumentModel> getAllDocuments() {
+        return documentRepository.findAll();
+    }
+
+    public DocumentModel addDocument(DocumentModel documentModel, Integer kbId) {
+
+        KbModel retKbModel = kbService.findOneKb(kbId);
+        DocumentModel retModel =null;
+        if (retKbModel == null) {
+            retModel= null;
```

Comment:
If a KB with the given ID doesn't exist should we return a 404? This currently returns `200 OK` without actually saving anything to the database.